### DISCUSSION TITLE
Guidance on combining samples.

### DISCUSSION THREAD

**Discussion by irwinwilliams** [microsoft/hack-together-teams#63] [Q&A] [answered]
Hi there 👋🏾!
Is there any guidance on how to combine samples, say proactive notifications and tabs? (Those are the ones we want to combine for our submission). 
I'm currently doing it manually, but as I'm about to run it, I'm almost certain I'll be all over the place trying to get them  to play nice.

**Comment by aycabas** [comment, marked as answer]
Hi @irwinwilliams, looking forward to your project submission 😊

There is a sample code that you can take a look as a reference, it shows how bot and a tab work together in a single project: https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/hello-world-bot-with-tab

There is also guidelines about configuring a tab in a bot: https://github.com/OfficeDev/TeamsFx/wiki/How-to-configure-Tab-capability-within-your-Teams-app

Or configuring a bot in a tab: https://github.com/OfficeDev/TeamsFx/wiki/How-to-configure-Bot-capability-within-your-Teams-app 

I hope these guidelines help you. Let me know if you have any other questions. Looking forward to see what you;ve build! 🥳

**Comment by irwinwilliams** [comment]
Nice! Thanks, @aycabas!
